cognite/client/_api/files.py

```diff
@@ -72,6 +72,54 @@ def __iter__(self) -> Generator[FileMetadata, None, None]:
         """
         return self.__call__()
 
+    def create(
+        self,
+        name: str,
+        external_id: str = None,
+        source: str = None,
+        metadata: Dict[str, Any] = None,
+        asset_ids: List[int] = None,
+        overwrite: bool = False,
+    ):
+        """Create file without uploading content.
+
+        Args:
+            external_id (str): External Id provided by client. Should be unique within the project.
+            name (str): No description.
+            source (str): The source of the file.
+            metadata (Dict[str, Any]): Customizable extra data about the file. String key -> String value.
+            asset_ids (List[int]): No description.
+            overwrite (bool): If 'overwrite' is set to true, and the POST body content specifies a 'externalId' field,
+                fields for the file found for externalId can be overwritten. The default setting is false.
+                If metadata is included in the request body, all of the original metadata will be overwritten.
+                File-Asset mappings only change if explicitly stated in the assetIds field of the POST json body.
+                Do not set assetIds in request body if you want to keep the current file-asset mappings.
+
+        Returns:
+            (FileMetaData, str): Tuple containing the file metadata and upload url of the created file.
+
+        Examples:
+
+            Create a file::
+
+                >>> from cognite.client import CogniteClient
+                >>> c = CogniteClient()
+                >>> res = c.files.create(name="my_file", metadata={"key": "value"})
```

Comment:
file_metadata, upload_url = c.files.create(...)

Reply:
For some reason, doing this causes an error in test_docstring_examples.py. It complains that there are not enough values to unpack (expected 2, got 0). Can you see why?

Reply:
Yes, it's because we mock the client so in the test .create() actually returns as MagickMock object. Just leave it like this!